### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by ericgaspar**
- Add new permissions

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

**Review by alexAubin — APPROVED** [review]
LGTM

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

File: scripts/install [hunk 1]
```diff
@@ -123,12 +120,14 @@ ynh_script_progression --message="Creating Drush alias..." --weight=2
...
+# cp -f "../conf/yoursite.aliases.drushrc.php" "$drush_aliasconfig"
 
-ynh_replace_string --match_string="__FINALPATH__" --replace_string="$final_path" --target_file="$drush_aliasconfig"
-ynh_replace_string --match_string="__APP__" --replace_string="$app" --target_file="$drush_aliasconfig"
-ynh_replace_string --match_string="__DOMAIN__" --replace_string="$domain" --target_file="$drush_aliasconfig"
-ynh_replace_string --match_string="__PATH_URL__" --replace_string="$path_url" --target_file="$drush_aliasconfig"
+# ynh_replace_string --match_string="__FINALPATH__" --replace_string="$final_path" --target_file="$drush_aliasconfig"
+# ynh_replace_string --match_string="__APP__" --replace_string="$app" --target_file="$drush_aliasconfig"
+# ynh_replace_string --match_string="__DOMAIN__" --replace_string="$domain" --target_file="$drush_aliasconfig"
+# ynh_replace_string --match_string="__PATH__" --replace_string="$path_url" --target_file="$drush_aliasconfig"
```
Comment (alexAubin) [inline scripts/install:128]:
```suggestion
```

Let's remove the lines if we ain't using them

File: scripts/install [hunk 2]
```diff
@@ -174,11 +173,13 @@ ynh_script_progression --message="Modifying a config file..." --weight=1
...
-ynh_replace_string --match_string="__DB_NAME__" --replace_string="$db_name" --target_file="$config_file"
-ynh_replace_string --match_string="__DB_USER__" --replace_string="$db_user" --target_file="$config_file"
-ynh_replace_string --match_string="__DB_PWD__" --replace_string="$db_pwd" --target_file="$config_file"
-ynh_replace_string --match_string="__FINALPATH__" --replace_string="$final_path" --target_file="$config_file"
+# cp ../conf/default.settings.php "$config_file"
+# ynh_replace_string --match_string="__DB_NAME__" --replace_string="$db_name" --target_file="$config_file"
+# ynh_replace_string --match_string="__DB_USER__" --replace_string="$db_user" --target_file="$config_file"
+# ynh_replace_string --match_string="__DB_PWD__" --replace_string="$db_pwd" --target_file="$config_file"
+# ynh_replace_string --match_string="__FINALPATH__" --replace_string="$final_path" --target_file="$config_file"
+
```
Comment (alexAubin) [inline scripts/install:181]:
```suggestion
```


Let's remove the lines if we ain't using them

**Review by alexAubin — APPROVED** [review]

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

**Comment by alexAubin** [comment]
!testme

**Pull request merged**